fairlearn/reductions/_moments/regression_moment.py

```diff
@@ -0,0 +1,134 @@
+from .utility_parity import UtilityParity
+from .moment import Moment,_LABEL,_GROUP_ID
+from .weighted_error_rate import WeightedErrorRate
+from .bounded_group_loss import SquareLoss
+import pandas as pd
+import numpy as np
+from itertools import repeat
+
+class CDF_DemographicParity(Moment):
```

Comment:
not sure how I feel about the underscore in the class name

If nobody else cares I'm willing to let it go, though :-)

Reply:
Steven told me that I should use this name, so I guess it is fine?

Reply:
Let's not resolve it unless @MiroDudik and @riedgar-ms also like that name.